src/lib/components/Swap/Summary/Details.tsx

```diff
@@ -23,31 +26,31 @@ function Detail({ label, value }: DetailProps) {
 }
 
 interface DetailsProps {
-  swap: Required<State>['swap']
   input: Currency
   output: Currency
 }
 
-export default function Details({
-  input: { symbol: inputSymbol },
-  output: { symbol: outputSymbol },
-  swap,
-}: DetailsProps) {
+export default function Details({ input, output }: DetailsProps) {
   const integrator = window.location.hostname
+
+  const [slippageTolerance] = useAtom(maxSlippageAtom)
```

Comment:
Pull these both from settingsAtom instead of the individual atoms.

Reply:
should integrator fee be moved to settings? (currenlty in swap) - not sure what that fee is

Reply:
Didn't realize that - I'd just leave it as is for now.